src/template/cloudflare-adapter.js

```diff
@@ -44,7 +45,13 @@ export async function handleRequest(event) {
         get: (target, prop) => target[prop] || target.PACKAGE.get(prop),
       }),
       storage: null,
+      attributes: {},
     };
+
+    // Initialize logger after context is created
+    // Logger configuration can be set via context.attributes.loggers
+    context.log = createCloudflareLogger(context.attributes.loggers, context);
```

Comment:
this would be better done for each call of the logger functions, so that you can add and remove loggers over the execution of a worker.